apps/web-staking/src/app/layout.tsx

```diff
@@ -29,6 +29,19 @@ export default function xRootLayout({
   // const initialState = cookieToInitialState(config, headers().get("cookie"));
   return (
     <html lang="en">
+      <head>
+        <meta name="title" property="og:title" content="Xai Staking App"/>
+        <meta name="description" property="og:description" content="Stake keys & esXai to earn rewards"/>
+        <meta name="image" property="og:image" content={`${process.env.SERVER_URL}/images/xai-preview.jpg`}/>
```

Comment:
Do we need to have `process.env.SERVER_URL`  here. In web-connect it seems we can pass in the image as binary. Then there should be a better way to manage this with NextJS, is there any context on why we need to have this ?
 
 This will be additional overhead having to add this variable to all pipelines.

Reply:
This should be fixed not to not required the server url!